app/controllers/course/assessment/submissions_controller.rb

```diff
@@ -108,17 +56,18 @@ def authorize_submission!
     end
   end
 
-  def load_or_create_answers
-    return unless @submission.attempting?
+  def load_service
```

Comment:
While possible, delegating actions doesn't seem like something the controller would do. It just looks funny?

I also rather not use around_action where possible because that adds another stack frame to the long list of stack frames. I use around_action only when I want to use an ensure block or the like.